dove/src/lib.rs

```diff
@@ -25,5 +25,5 @@ pub mod manifest;
 pub mod stdout;
 /// for tests
 pub mod tests_helper;
```

Comment:
Just use `#[doc(hidden)]` or/and `#![doc(hidden)]` in the module `tests_helper`.

Reply:
Done.